src/components/Footer.tsx

```diff
@@ -106,7 +92,7 @@ const styles = {
             xs: 5,
             sm: 5,
         },
-        bgcolor: "#4dabf5",
+        bgcolor: "#4d4d4d",
```

Comment:
For styles it would be better to use the theme variables. This way they will update automatically with theme changes. Also, we then only need to change colours in the theme file and it will update across the whole site